crates/sui-transaction-checks/src/deny.rs

```diff
@@ -175,9 +175,8 @@ fn check_package_dependencies(
             Command::MoveCall(call) => {
                 let package = package_store.get_package_object(&call.package)?.ok_or(
                     SuiError::UserInputError {
-                        error: UserInputError::ObjectNotFound {
-                            object_id: call.package,
-                            version: None,
+                        error: UserInputError::DependentPackageNotFound {
+                            package_id: call.package,
                         },
                     },
```

Comment:
I think we should just skip the command in this case? This is just for `deny` and we still will check the objects later